### ISSUE TITLE
django-rest-knox for token-based auth

### ISSUE THREAD

**Issue by spinxi** [sunscrapers/djoser#809]
django-rest-knox has better token based auth protection, so is there any way to use this library with Djoser for token auth?

**Comment by gawry** [comment]
Yes, there is. its kinda hidden in the docs but there is a tip there. You basically need to setup the token model and use the authentication from rest knox in DRF config

https://djoser.readthedocs.io/en/latest/settings.html#token-model
